### PR TITLE
Fix rz-pipe.go building on CI

### LINKED ISSUE #42: Failure to build `rz-pipe.go`
```
Run go get github.com/rizinorg/rz-pipe/go
go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
```

### PR DESCRIPTION
**Pull request by XVilka**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Fixes https://github.com/rizinorg/rz-pipe/issues/42

**Test plan**

CI is green

### CODE REVIEW

**Review by ret2libc — COMMENTED** [review]
Didn't we already fix this somewhere?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -760,7 +760,7 @@ jobs:
           pip install -U rzpipe
       - name: Install rz-pipe-go
         run: |
-          go get github.com/rizinorg/rz-pipe/go
+          go install github.com/rizinorg/rz-pipe/go@latest
```
Comment (wargio) [inline .github/workflows/ci.yml]: do a `cd example` before

**Comment by kazarmy** [comment]
> Didn't we already fix this somewhere?

You mean [this fix](https://github.com/rizinorg/rizin/pull/3063/commits/2934e3129bcc8815ea2d6d583ed34150bf6b5b91)? ... this is a different problem

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kazarmy) [reply .github/workflows/ci.yml]: ok 25fba033ee3e42c12f0f461387ae98e6b0d1cd1e

**Review by kazarmy — APPROVED** [review]

**Review by wargio — APPROVED** [review]

**Pull request merged**